Class 2 Exercises and Homework.js

```diff
@@ -4,41 +4,69 @@
 // - lastName
 // - 'favorite food'
 // - bestFriend (object with the same 3 properties as above)
-
+// Myself object
+const koteshObject = {
+  firstName: 'Kotesh',
+  lastName: 'Boda',
+  'favorite food': 'Mediterranean',
+  
+  bestFriend: {
+    firstName: 'Brad',
+    lastName: 'Pitt',
+    'favorite food': 'Mexican',  
+  },
+};
 
 // 2. console.log best friend's firstName and your favorite food
-
+console.log(`${koteshObject.firstName} best friend's first name is: `, koteshObject.bestFriend.firstName);
+console.log(`${koteshObject.firstName} favorite food is: `, koteshObject['favorite food']);
 
 // 3. Create an array to represent this tic-tac-toe board
 // -O-
 // -XO
 // X-X
-
+const ticTacToeArray = [['-', 'O', '-'], ['-', 'X', 'O'], ['X', '-', 'X']];
 
 // 4. After the array is created, 'O' claims the top right square.
-// Update that value.
-
+// Update the top right square value.
+ticTacToeArray[0][2] = 'O';
 
 // 5. Log the grid to the console.
-
+console.log('Tic Tac Toe Array first row: ', ticTacToeArray[0]);
+console.log('Tic Tac Toe Array second row: ', ticTacToeArray[1]);
+console.log('Tic Tac Toe Array thrid row: ', ticTacToeArray[2]);
 
 // 6. You are given an email as string myEmail, make sure it is in correct email format.
 // Should be 1 or more characters, then @ sign, then 1 or more characters, then dot, then one or more characters - no whitespace
 // i.e. foo@bar.baz
 // Hints:
 // - Use rubular to check a few emails: https://rubular.com/
 // - Use regexp test method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test
+const myEmail = 'test@test.com';
```

Comment:
Try using a few "bad" emails to test this code. I got `true` when using "test@testcom".